src/acl.c

```diff
@@ -3204,7 +3204,7 @@ void addReplyCommandCategories(client *c, struct serverCommand *cmd) {
 }
 
 /* AUTH <password>
- * AUTH <username> <password> (Redis >= 6.0 form)
+ * AUTH <username> <password> (Redis OSS >= 6.0 form)
```

Comment:
IANAL - but it probably wouldn't be an issue to leave it.

However - with the license changes, Redis was clear on what naming they wanted moving forward for "Redis Community Edition" vs "Redis OSS": https://redis.com/blog/redis-adopts-dual-source-available-licensing/

> We have referred to versions of our products as either open source (OSS), Enterprise, or Cloud. Redis v7.2 and prior releases will continue to be called Redis OSS. Starting with Redis v7.4, we will refer to the open, freely available versions as Redis Community Edition. Both the RSALv2 and SSPL v1 licenses provide open and free access to the underlying source code of Redis Community Edition. However, it does not meet the definition of open source as defined by OSI, and that is why we will refer to the products as the “community edition” rather than the “open source” edition, as we did previously. There are many references to open source on our websites, and we are actively working to clarify these language changes in the coming weeks.

To me - it makes sense to align our nomenclature. But yeah, happy to keep this open for discussion.

Reply:
@zuiderkwast - anyone in particular you want to involve in the discussion?

Reply:
Not really. You're right. Let's use Redis OSS. :+1: